src/lib/Output/Generator/Json/FieldTypeHashGenerator.php

```diff
@@ -166,7 +166,11 @@ private function generateObjectValue($parent, object $value)
             $value = null;
         }
 
-        return $this->generateValue($parent, $value);
+        if (is_array($value)) {
+            return $this->generateArrayValue($parent, $value);
+        }
+
+        return $value;
```

Comment:
Hmm... Expected at the end
```php
return $this->generateValue($parent, $value);
```
and just different behavior for an array, given the pre-existing code.